### ISSUE TITLE
The "Enterprise Manager" is not available

### ISSUE THREAD

**Issue by pilot513** [oracle/docker-images#1419]
When run Docker container builded from 12.2.0.1
on OEL 7.7 (./buildDockerImage.sh -v 12.2.0.1 -e)
and run it:
`docker run --name oracle12-ee -p 1521:1521 -p 5500:5500 -e ORACLE_SID=ORCLCDB -e ORACLE_PDB=ORCLPDB1 -v /home/oracle/oradata:/opt/oracle/oradata oracle/database:12.2.0.1-ee`
the site https://ip:5500/em not available, but all mapped ports listened:
`987139b3af60        oracle/database:12.2.0.1-ee   "/bin/sh -c 'exec $O…"   About an hour ago   Up About an hour (healthy)   0.0.0.0:1521->1521/tcp, 0.0.0.0:5500->5500/tcp   oracle12-ee

tcp    LISTEN     0      128    [::]:5500               [::]:*
tcp    LISTEN     0      128    [::]:1521               [::]:*`

The connection on 1521 work propetly.

**Comment by mriccell** [comment]
No Enterprise Manager is not available in these Docker images.

**Comment by andywarduk** [comment]
I fixed this by following advice in doc ID 2680248.1 on Oracle support:

```
docker exec -it racnode1 /bin/bash
su - oracle
setfacl -R -m u:grid:rwx <XDB wallet location>
eg. setfacl -R -m u:grid:rwx /u01/app/oracle/product/19.3.0/dbhome_1/admin/ORCLCDB/xdb_wallet
```
